charts/seed-monitoring/charts/prometheus/templates/blackbox-exporter-config.yaml

```diff
@@ -21,6 +21,6 @@ data:
             Accept-Language: "en-US"
           tls_config:
             ca_file: "/var/run/secrets/blackbox/ca.crt"
-            cert_file: "/var/run/secrets/blackbox/kube-apiserver.crt"
-            key_file: "/var/run/secrets/blackbox/kube-apiserver.key"
+            cert_file: "/var/run/secrets/blackbox/prometheus.crt"
+            key_file: "/var/run/secrets/blackbox/prometheus.crt"
```

Comment:
Did you test wether this PR works actually? You have specified the certificate at the `key_file` variable. From my PoV it cannot work.
If it does work then I don't understand what this `tls_config` is used for.

Please fix.